dashboard/test/models/script_test.rb

```diff
@@ -1549,36 +1549,6 @@ class CanViewVersion < ActiveSupport::TestCase
     assert_equal("Waiting for review", feedback2_result[:reviewStateLabel])
   end
 
-  # This test checks that all categories that may show up in the UI have
-  # translation strings.
-  test 'all visible categories have translations' do
```

Comment:
just confirming my understanding that this statement is now enforced by the not null requirement here? https://github.com/code-dot-org/code-dot-org/blob/b40e5e08b5218068e5d4da51143fe29b4c48975c/dashboard/app/models/course_offering.rb#L7